addons/mail/static/src/core/common/suggestion_service.js

```diff
@@ -190,6 +198,42 @@ export class SuggestionService {
         };
     }
 
+    searchRoleSuggestions(cleanedSearchTerm, sort) {
```

Comment:
Why do we have a sort param? It seems to always be called with true.

Reply:
will remove it in this PR